misc-tools/dj_make_chroot.in

```diff
@@ -369,7 +369,7 @@ chroot "$CHROOTDIR" /bin/sh -c "chmod a-s /usr/bin/wall /usr/bin/newgrp \
 # Disable root account
 sed -i "s/^root::/root:*:/" "$CHROOTDIR/etc/shadow"
 
-# Create a dummy file to test that in the judging environment no
+# Create a placeholder file to test that in the judging environment no
```

Comment:
This is not a placeholder, since there's nothing to be put in its place.

Reply:
By removing the placeholder the sentence still works.